index.html

```diff
@@ -11,10 +11,8 @@
 <body>
     <h1>Welcome to Betina's site</h1>
     <main>
```

Comment:
- Please use descriptive commit messages in your future commits - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).

![image](https://user-images.githubusercontent.com/47494310/230606555-5536b883-8dca-4bf0-aac9-46e020e0ee8b.png)